packages/mobile-top-nav/README.md

```diff
@@ -22,65 +23,55 @@ export default IAMobileTopNav;
 </script>
 
 <style>
-  scrubber-bar {
-    --thumbColor: white;
-    --thumbBorder: 1px solid black;
-    --trackFillColor: blue;
-    --trackColor: purple;
-    --trackBorder: 1px solid black;
-    --trackBorderRadius: 5px;
-    --trackHeight: 10px;
-    --thumbDiameter: 20px;
-    --scrubberBarHeight: 20px;
-    --thumbBorderRadius: 10px;
-    --webkitThumbTopMargin: -6px;
+  /* Defaults */
+  topnav-element {
+    --white: #fff;
```

Comment:
Are your CSS variables named `white` and `grey20`? You want them to be more semantically meaningful like `titleColor` and `bodyColor` that can then be set to `#fff` or `#333`.

Reply:
I think Kanchan was modeling the variable names after the existing Less names in Petabox.